shenyu-client/shenyu-client-grpc/src/main/java/org/apache/shenyu/client/grpc/GrpcClientBeanPostProcessor.java

```diff
@@ -122,17 +124,31 @@ private void handler(final Object serviceBean) {
             LOG.error(String.format("grpc SERVICE_NAME can not found: %s", classes));
             return;
         }
+        ShenyuGrpcClient grpcClassAnnotation = AnnotationUtils.findAnnotation(clazz, ShenyuGrpcClient.class);
+        String basePath = Objects.nonNull(grpcClassAnnotation)
+                ? StringUtils.defaultIfBlank(grpcClassAnnotation.path(), "")
+                : "";
```

Comment:
`Optional.ofNullable(grpcClassAnnotation).map(annotation->StringUtils.defaultIfBlank(grpcClassAnnotation.path(), "")).orElse("")` may be better.

Reply:
Good practice. I'll change it